alwaysontop/render.js

```diff
@@ -214,6 +244,7 @@ class AlwaysOnTop {
             api: this._api,
             onload: this._updateLargeVideoSrc,
             onbeforeunload: () => {
+                this._api.emit(ALWAYSONTOP_WILL_CLOSE);
```

Comment:
Currently we don't have any events or public interfaces in the IFrame API related to the always on top. Also I think this will fit better to the jitsi-meet-electron-utils API.

Could you please not emit this trough the IFrame API instance but handle it somehow in the jitsi-meet-electron-utils project? 
One option that I can think of is to pass an "always on top will close" callback to `setupAlwaysOnTopRender`. Another one will be to return an `Emitter` from the `setupAlwaysOnTopRender` which can be used to emit the `ALWAYSONTOP_WILL_CLOSE` event.  Maybe there are even better options...

Also could you please add some information about this in the README.md?

Reply:
Changed the initial piggybacking on the external api to an EventEmitter instance. Added some extra fixes for corner cases aswell.